wiki/Client/Beatmap_editor/Song_setup/en.md

```diff
@@ -1,238 +1,121 @@
 
-Set the volume of the sample set globally.
-
-#### Test Sample Set
-
-Clicking on those buttons to test out how the sample set sounds. Four buttons are provided, namely: `Normal`, `Whistle`, `Finish`, and `Clap`. These sounds are additive; `Whistle` is a combination of `Normal` and `Whistle`.
-
-### Misc. Toggles
-
-The `Samples match playback rate (for fully-hitsounded maps)` checkbox will adjust the hitsounds' pitch accordingly to match the speed modifiers (both in the editor and in-game). Unchecking this will allow the hitsounds to play at its own preset pitch.
+| Control | Effect |
+| :-- | :-- |
+| Samplesets: `Normal/Soft/Drum` | Select between different built-in [samplesets](/wiki/Beatmapping/Sampleset). |
+| `Enable custom overrides` | Use [custom hitsounds](/wiki/Guides/Custom_sample_overrides) in place of the default ones. |
+| `Sample Volume` | Tweak the volume of hitsounds globally. |
+| Hitsound buttons | Play hitsounds to be used. |
+| `Samples match playback rate` | Alter hitsounds' pitch and tempo when the beatmap speed changes (both in the editor and in-game). |
 
 ## Colours
 
 ![Screenshot of the Colours tab](img/SS_Colours.jpg "Colours tab")
 
-Configure the combo colours and playfield's background colour.
-
-### Hit circle/Slider Combos
-
-Configure the custom combo colour set. Combo colouring is an important aspect to a beatmap's aesthetic value.
-
-Clicking a combo colour or clicking on `New Combo Colour` opens up the operating system's colour picker (as shown in the image; Windows' colour picker). Choose colours that compliment the background, not camouflage into the background.
-
-At most, eight combo colours can be selected, although most beatmaps commonly use four.
-
-The `Enable Custom Colours` checkbox must be checked to enable the custom combo colour set above. Unchecking it will return to either the default combo colour settings or the player's skin combo colour settings. It is generally recommended to keep it checked so the beatmap can have its own unique identity in terms of combo colours. It can be safely unchecked for osu!taiko-specific and osu!mania-specific beatmaps.
+`Colours` allows for configuring the [combo colours](/wiki/Beatmapping/Combo_colour) and the playfield's background colour, in case the beatmap is missing the [background](/wiki/Beatmap/Background) image or video.
 
-Clicking the `Remove Combo Colour` will remove the last defined combo colour.
+During gameplay, the colours of hit objects will cycle in the specified order, changing on every [new combo](/wiki/Beatmapping/New_combo). As a result, it's important to not only synchronize combos with the song, but also to pick combo colours that compliment the background and make the hit objects well-readable.
```

Comment:
```suggestion
During gameplay, the colours of hit objects will cycle in the specified order by default, changing on every [new combo](/wiki/Beatmapping/New_combo). As a result, it's important to not only synchronize combos with the song, but also to pick combo colours that compliment the background and make the hit objects well-readable.
```
subtle implication that this can be changed, idk if it's worth detailing here tho